frontend/src/styles/global.scss

```diff
@@ -25,6 +25,22 @@ body strong {
     font-weight: 600;
 }
 
+// Disable default styling for ul...
+ul {
+    margin-bottom: 0;
+    list-style: none;
+    padding-left: 0;
+}
+
+// ... unless in a <p> element
+p {
+    ul {
+        margin-bottom: 1rem;
+        list-style: initial;
+        padding-left: 1.5rem;
+    }
+}
```

Comment:
I think [`ul.list-disc`](https://tailwindcss.com/docs/list-style-type) followed by `.space-y-2` or whatnot is a better solution than this _illegal_ solution? :)